sdk/messaging/azservicebus/internal/atom/manager_common_test.go

```diff
@@ -37,13 +38,13 @@ func newFakeResponse(statusCode int, status string, contents string) *http.Respo
 
 func TestResponseError(t *testing.T) {
 	resp := newFakeResponse(http.StatusConflict, "statusString", "")
-	require.Contains(t, NewResponseError(resp).Error(), "statusString")
+	require.Contains(t, runtime.NewResponseError(resp).Error(), "statusString")
```

Comment:
Might be good to test that the fields in the `ResponseError` are properly populated.

Reply:
Just realized, this is a silly test since now I'm just testing the azcore.ResponseError, which is already tested in its own package. Fixed by deleting this test.